zokrates_core/src/typed_absy/mod.rs

```diff
@@ -1243,6 +1263,12 @@ impl<'ast, T> From<TypedExpression<'ast, T>> for StructExpression<'ast, T> {
     }
 }
 
+impl<'ast, T> From<TypedExpression<'ast, T>> for TypedExpressionList<'ast, T> {
+    fn from(_: TypedExpression<'ast, T>) -> TypedExpressionList<'ast, T> {
+        unreachable!()
```

Comment:
This `From` seems a bit unclear at first glance, maybe you should leave a comment here on why is it needed and unreachable?